mule-module-apikit/src/main/java/org/mule/module/apikit/UrlUtils.java

```diff
@@ -114,7 +114,7 @@ public static String rewriteBaseUri(String raml, String baseSchemeHostPort)
 
     public static String replaceBaseUri(String raml, String newBaseUri)
     {
-        return replaceBaseUri(raml, "https?://.*$", newBaseUri);
+        return replaceBaseUri(raml, "(https?://.*$|\\/.*$)", newBaseUri);
```

Comment:
Is it possible for a relative baseUri to not start with `/`?
e.g.:
- `api`
- `../api`